airflow/triggers/temporal.py

```diff
@@ -32,9 +32,13 @@ class DateTimeTrigger(BaseTrigger):
     a few seconds.
 
     The provided datetime MUST be in UTC.
+
+    :param moment: when to yield event
+    :param exit_task: whether the trigger should mark the task successful after time condition
+        reached or resume the task
     """
 
-    def __init__(self, moment: datetime.datetime):
+    def __init__(self, moment: datetime.datetime, exit_task=False):
```

Comment:
Can be a good idea to make the new argument kwarg-only

Reply:
sure